src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
naming of interfaces is better when it's noun or adjective
verbs are usually refer to methods naming
```suggestion
public interface AreaCalculator {
```